### DISCUSSION TITLE
Disable logging entire JSON output

### DISCUSSION THREAD

**Discussion by NaxAlpha** [bentoml/BentoML#1864] [Q&A]
I want to ask if it is possible to disable printing the entire JSON output on each request as some requests contain JSON of size ~100MB so sometimes printing it a lot causes slow down or crash etc.

**Comment by parano** [comment]
Hi @NaxAlpha, yes it is possible, see the logging topic in the advanced guide section: https://docs.bentoml.org/en/latest/guides/logging.html

**Reply by NaxAlpha** [reply]
Thanks a lot for sharing that. It seems to me that documentation is not very clear on how I can achieve what I mentioned in my post. Can you share a short configuration on how to just disable printing the output but I want to keep the fastapi/flask output intact.
Thanks

**Comment by gregd33** [comment]
I have a similar issue/request and agree the documentation isn't clear. I've tried adding a custom yaml to the Docker image and setting `BENTOML_CONFIG=/bentoml_configuration.yml` and this yaml sets log level to WARNING (and prediction log is INFO level). So it should work according to the docs but doesn't.

**Reply by zfallahnejad** [reply]
I have the same issue. I find the documentation unclear as well. I tried to add a config file but it didn't work.
